### ISSUE TITLE
Missing data in eds_subjects_geographic

### ISSUE THREAD

**Issue by drh-stanford** [ebsco/edsapi-ruby#43]
We're not seeing any data for the `eds_subjects_geographic` field. Here's an example -- the location data is in the `eds_subjects` field rather than the `eds_subjects_geographic` field.

```yml
--- !ruby/hash:ActiveSupport::HashWithIndifferentAccess
eds_accession_number: edsgcl.299362683
eds_database_id: edsgcc
eds_database_name: Canada In Context
eds_access_level: '6'
eds_relevancy_score: '861'
eds_title: With fins like these, who needs anemones? Shark hunters seek big fish to
  lure visitors to Toronto's new aquarium
eds_source_title: The Toronto Star (Toronto, Ontario)
eds_composed_title: The Toronto Star (Toronto, Ontario). August 12, 2012, A1.
eds_other_titles: 
eds_abstract: 'A team of shark hunters has taken to the sea in search of Toronto''s
  newest downtown spectacle.   The hunters: a squadron of marine biologists and shark
  specialists.  [...]'
eds_authors: ''
eds_author_affiliations: 
eds_subjects:
- Ripley Entertainment Inc.
- Canada
- United States
- South Carolina
- Ontario
eds_subjects_geographic: 
eds_subjects_person: 
eds_author_supplied_keywords: 
eds_notes: 
eds_languages:
- English
eds_page_count: 
eds_page_start: 
eds_physical_description: 
eds_publication_type: News
eds_publication_type_id: newspaperArticle
eds_publication_date: '2012-08-12'
eds_publication_year: '2012'
eds_publication_info: 
eds_document_type: 
eds_document_doi: 
eds_document_oclc: 
eds_issn_print: '03190781'
eds_issns:
- '03190781'
eds_isbn_print: 
eds_isbns_related: 
eds_isbn_electronic: 
eds_isbns: []
eds_series: 
eds_volume: 
eds_issue: 
eds_covers: []
eds_cover_thumb_url: 
eds_cover_medium_url: 
eds_fulltext_word_count: 0
eds_result_id: 1
eds_plink: http://search.ebscohost.com/login.aspx?direct=true&site=eds-live&db=edsgcc&AN=edsgcl.299362683
eds_html_fulltext: 
eds_images: []
eds_all_links:
- !ruby/hash:ActiveSupport::HashWithIndifferentAccess
  url: http://library.stanford.edu/sfx?genre=article&atitle=With%20fins%20like%20these%2C%20who%20needs%20anemones%3F%20Shark%20hunters%20seek%20big%20fish%20to%20lure%20visitors%20to%20Toronto%27s%20new%20aquarium&title=The
    Toronto Star (Toronto, Ontario)&issn=03190781&isbn=&volume=&issue=&date=20120812&aulast=&spage=&pages=&sid=EBSCO:Canada%20In%20Context:edsgcl.299362683
  label: Check SFX for full text
  icon: http://library.stanford.edu/sfx/img/sfxmenu/SFXFinal.gif
  type: customlink-fulltext
eds_fulltext_links:
- !ruby/hash:ActiveSupport::HashWithIndifferentAccess
  url: http://library.stanford.edu/sfx?genre=article&atitle=With%20fins%20like%20these%2C%20who%20needs%20anemones%3F%20Shark%20hunters%20seek%20big%20fish%20to%20lure%20visitors%20to%20Toronto%27s%20new%20aquarium&title=The
    Toronto Star (Toronto, Ontario)&issn=03190781&isbn=&volume=&issue=&date=20120812&aulast=&spage=&pages=&sid=EBSCO:Canada%20In%20Context:edsgcl.299362683
  label: Check SFX for full text
  icon: http://library.stanford.edu/sfx/img/sfxmenu/SFXFinal.gif
  type: customlink-fulltext
eds_non_fulltext_links: []
id: edsgcc__edsgcl_299362683
eds_publication_id: 
eds_publication_is_searchable: 
eds_publication_scope_note: 
eds_fulltext_link: !ruby/hash:ActiveSupport::HashWithIndifferentAccess
  id: edsgcc__edsgcl_299362683
  links:
  - !ruby/hash:ActiveSupport::HashWithIndifferentAccess
    url: http://library.stanford.edu/sfx?genre=article&atitle=With%20fins%20like%20these%2C%20who%20needs%20anemones%3F%20Shark%20hunters%20seek%20big%20fish%20to%20lure%20visitors%20to%20Toronto%27s%20new%20aquarium&title=The
      Toronto Star (Toronto, Ontario)&issn=03190781&isbn=&volume=&issue=&date=20120812&aulast=&spage=&pages=&sid=EBSCO:Canada%20In%20Context:edsgcl.299362683
    label: Check SFX for full text
    icon: http://library.stanford.edu/sfx/img/sfxmenu/SFXFinal.gif
    type: customlink-fulltext
marcfield: 
```

**Comment by drh-stanford** [comment]
I found a record that has geographic subjects: http://localhost:3000/article/aph__123200654

```yml
--- !ruby/hash:ActiveSupport::HashWithIndifferentAccess
eds_result_id: 1
eds_plink: http://search.ebscohost.com/login.aspx?direct=true&site=eds-live&db=aph&AN=123200654
eds_accession_number: '123200654'
eds_database_id: aph
eds_database_name: Academic Search Premier
eds_access_level: '6'
eds_relevancy_score: 
id: aph__123200654
eds_title: 'Environmental policy and climate change vulnerability in the Maldives:
  from the ''lexicon of risk'' to social response to change.'
eds_source_title: Island Studies Journal
eds_composed_title: <searchLink fieldCode="JN" term="%22Island+Studies+Journal%22">Island
  Studies Journal</searchLink>. 2017, Vol. 12 Issue 1, p53-70. 18p.
eds_other_titles: 
eds_abstract: The climate change vulnerability discourse in the Maldives coexists
  with a pervasive set of critical environmental factors of significance to the socio-environmental
  systems of small peripheral islands. This implies the need to strike a balance between
  global challenges associated with environmental processes at the supra-national
  scale and the adjustments and strategies implemented at the local scale in response
  to change. The current paper offers a discussion of this dialectic, in reference
  to both the broader contemporary debate in island studies, and the political and
  environmental context of the Maldives. We first outline the international scenario,
  and then go on, in the second part of the paper, to provide a reading of environmental
  policy on these islands. We argue that emphasizing the country's environmental vulnerability
  has reinforced a 'lexicon of risk' within the environmental discourse and that,
  in recent years, this narrative has been one of the main forces driving the construction
  of contemporary Maldivian 'nation-ness'. [ABSTRACT FROM AUTHOR]
eds_authors:
- Malatesta, Stefano
- di Friedberg, Marcella Schmidt
eds_authors_composed: "&lt;searchLink fieldCode=&quot;AR&quot; term=&quot;%22Malatesta%2C+Stefano%22&quot;&gt;Malatesta,
  Stefano&lt;/searchLink&gt;&lt;relatesTo&gt;1&lt;/relatesTo&gt;&lt;i&gt; stefano.malatesta@unimib.it&lt;/i&gt;&lt;br
  /&gt;&lt;searchLink fieldCode=&quot;AR&quot; term=&quot;%22di+Friedberg%2C+Marcella+Schmidt%22&quot;&gt;di
  Friedberg, Marcella Schmidt&lt;/searchLink&gt;&lt;relatesTo&gt;1&lt;/relatesTo&gt;&lt;i&gt;
  marcella.schmidt@unimib.it&lt;/i&gt;"
eds_author_affiliations: 
eds_subjects:
- MALDIVES
- CLIMATE change
- ENVIRONMENTAL policy
- ISLANDS
- SOCIETAL reaction
eds_subjects_geographic: "&lt;searchLink fieldCode=&quot;DE&quot; term=&quot;%22MALDIVES%22&quot;&gt;MALDIVES&lt;/searchLink&gt;"
eds_subjects_person: 
eds_subjects_genre: 
eds_author_supplied_keywords: 
eds_notes: 
eds_subset: 
eds_languages:
- English
eds_page_count: '18'
eds_page_start: '53'
eds_physical_description: 
eds_publication_type: Academic Journal
eds_publication_type_id: academicJournal
eds_publication_date: '2017-05-01'
eds_publication_year: '2017'
eds_publication_info: 
eds_document_type: 
eds_document_doi: 10.24043/isj.5
eds_document_oclc: 
eds_issn_print: '17152593'
eds_issns:
- '17152593'
eds_isbn_print: 
eds_isbns_related: 
eds_isbn_electronic: 
eds_isbns: []
eds_series: 
eds_volume: '12'
eds_issue: '1'
eds_covers: []
eds_cover_thumb_url: 
eds_cover_medium_url: 
eds_fulltext_word_count: 0
eds_html_fulltext: 
eds_images: []
eds_all_links:
- !ruby/hash:ActiveSupport::HashWithIndifferentAccess
  url: detail
  label: PDF Full Text
  icon: PDF Full Text Icon
  type: pdf
eds_fulltext_links:
- !ruby/hash:ActiveSupport::HashWithIndifferentAccess
  url: detail
  label: PDF Full Text
  icon: PDF Full Text Icon
  type: pdf
eds_non_fulltext_links: []
eds_code_naics: 
eds_abstract_supplied_copyright: "&lt;i&gt;Copyright of Island Studies Journal is
  the property of Island Studies Journal and its content may not be copied or emailed
  to multiple sites or posted to a listserv without the copyright holder&#39;s express
  written permission. However, users may print, download, or email articles for individual
  use. This abstract may be abridged. No warranty is given about the accuracy of the
  copy. Users should refer to the original published version of the material for the
  full abstract.&lt;/i&gt; (Copyright applies to all Abstracts.)"
eds_descriptors: 
eds_publication_id: 
eds_publication_is_searchable: 
eds_publication_scope_note: 
eds_fulltext_link: !ruby/hash:ActiveSupport::HashWithIndifferentAccess
  id: aph__123200654
  links:
  - !ruby/hash:ActiveSupport::HashWithIndifferentAccess
    url: detail
    label: PDF Full Text
    icon: PDF Full Text Icon
    type: pdf
marcfield: 
```

**Comment by bmckinney** [comment]
looks like geo subjects can appear in 2 ways:

```
<Name>Subject</Name>
<Label>Subject Geographic</Label>
<Group>Su</Group>
```

and

```
<Name>SubjectGeographic</Name>
<Label>Geographic Terms</Label>
<Group>Su</Group>
```

**Comment by bmckinney** [comment]
should be fixed in 0.3.14.pre, see: https://github.com/ebsco/edsapi-ruby/commit/cf95cf764a7a1c5610597cb708f28a6b6234bab8

**Comment by efrierson** [comment]
This was resolved per Bill's comment.  Closing.  Please re-open if this continues to be a problem.
